pkg/controller/linstorcsidriver/linstorcsidriver_controller.go

```diff
@@ -367,6 +376,7 @@ var (
 	MountPropagationBidirectional = corev1.MountPropagationBidirectional
 	HostPathDirectoryOrCreate     = corev1.HostPathDirectoryOrCreate
 	HostPathDirectory             = corev1.HostPathDirectory
+	DefaultHealthPort             = 9808
```

Comment:
Where it comes from?
Standard linstor-csi have no any health checks implemented yet, and does not listening on 9808 port

Reply:
This is part of the [`csi-livenessprobe`](https://github.com/kubernetes-csi/livenessprobe) container

Reply:
Ah, sorry.
I was confused by the livenessProbe section location for another container🙂